### PR TITLE
Updated model tab for day2 UI (SOC-8879)

### PR DESCRIPTION
**Pull request by guglo**

### CODE REVIEW

File: xml/day2_ui_documentation.xml [hunk 1]
```diff
@@ -224,6 +224,85 @@
...
+   <imageobject role="fo">
+    <imagedata fileref="clm_admin_services_model_2.png" width="75%"/>
+   </imageobject>
+   <imageobject role="html">
+    <imagedata fileref="clm_admin_services_model_2.png"/>
+   </imageobject>
+  </mediaobject>
+ </figure>
+ <para>
+ Click <guimenu>Update<guimenu> to start deployment. The status of those changes will be
```
Comment (jeremy-moffitt) [inline xml/day2_ui_documentation.xml]: the second <guimenu> here is the cause of the validation errors, its supposed to be `</guimenu>` ... missing the forwardslash

Comment (csymons-suse) [reply xml/day2_ui_documentation.xml]: I will edit and do a new PR this morning.

**Comment by csymons-suse** [comment]
superseded by https://github.com/SUSE-Cloud/doc-cloud/pull/993

**Comment by csymons-suse** [comment]
superseded by https://github.com/SUSE-Cloud/doc-cloud/pull/993

**Pull request closed without merging**